### PR TITLE
Get requests

### PR DESCRIPTION
**Pull request by DennisMaas**

### CODE REVIEW

File: frontend/src/components/OverView.js [hunk 1]
```diff
@@ -4,20 +4,14 @@ import { Container, Grid } from '@material-ui/core';
...
-          <Grid item xs={12} sm={6} md={4} lg={3}>
-            <PlaceCard
-              key={placeData.id}
-              type={placeData.type}
-              title={placeData.title}
-              street={placeData.street}
-              primaryPictureUrl={placeData.primaryPictureUrl}
-            />
+        {places?.map((placeData) => (
+          <Grid item xs={12} sm={6} md={4} lg={3} key={placeData.id}>
```
Comment (benjamin-ranft) [inline frontend/src/components/OverView.js:13]: Why not with styled components instead of inline?

Comment (benjamin-ranft) [reply frontend/src/components/OverView.js:13]: ah because of Material UI right?

**Comment by benjamin-ranft** [comment]
LGTM but your build frontend CI fails?

**Review by benjamin-ranft — APPROVED** [review]

File: backend/src/main/java/de/dennismaas/thegramfworkingtitle/controller/PlaceController.java [hunk 2]
```diff
@@ -19,6 +19,7 @@ public PlaceController(PlaceService placeService){
         this.placeService = placeService;
     }
 
+    @CrossOrigin
```
Comment (christophersiem) [inline backend/src/main/java/de/dennismaas/thegramfworkingtitle/controller/PlaceController.java:22]: do you really need the _CrossOrigin Annotation_ here?

File: frontend/src/components/inDepthDetailView/InDepthDetailView.js [hunk 3]
```diff
@@ -42,14 +45,13 @@ const useStyles = makeStyles((theme) => ({
...
-  primaryPictureUrl
-) {
+export default function InDepthDetailView() {
   const classes = useStyles();
-  return (
+  const { id } = useParams();
+  const { places } = useContext(PlacesContext);
+  const placeData = places.find((place) => place.id === id);
+
+  return !placeData ? null : (
```
Comment (christophersiem) [inline frontend/src/components/inDepthDetailView/InDepthDetailView.js:54]: works fine, but I'd use the "logical && Operator" here..
`{placeData  && ...`

[https://reactjs.org/docs/conditional-rendering.html](https://reactjs.org/docs/conditional-rendering.html)

**Review by christophersiem — COMMENTED** [review]
🦜 well done

**Pull request merged**